app/javascript/packs/document-capture.tsx

```diff
@@ -120,6 +120,8 @@ try {
   parsedUsStatesTerritories = JSON.parse(usStatesTerritories);
 } catch (e) {}
 
+const extendSessionIfNotInPersonProofing = optedInToInPersonProofing ? () => null : extendSession;
```

Comment:
I think `optedInToInPersonProofing` a string, you have to check if it is strict equal to string `'true'`?

```suggestion
const extendSessionIfNotInPersonProofing = optedInToInPersonProofing === 'true' ? () => null : extendSession;
```